en/core-libraries/toc-utilities.rst

```diff
@@ -1,22 +0,0 @@
-Utilities
-#########
```

Comment:
I hadn't thought about that...

I tried to create empty files with ``..include`` for old path to keep the old path. The only problem is that we can access the same page with 2 urls....